### PR TITLE
make /graph as default endpoint for Cincinnati

### PR DESCRIPTION
**Pull request by PratikMahajan**
This PR makes the endpoint `/graph` the default endpoint for Cincinnati. 
Cincinnati still supports the older `/v1/graph` endpoint.

### CODE REVIEW

**Comment by wking** [comment]
[Failed install][1].

/retest

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati/537/pull-ci-openshift-cincinnati-master-customrust-e2e/1441080958508863488

File: policy-engine/src/main.rs [hunk 1]
```diff
@@ -117,9 +117,14 @@ async fn main() -> Result<(), Error> {
...
+                // keeping this for backward compatibility
                 actix_web::web::resource(&format!("{}/v1/graph", app_prefix))
                     .route(actix_web::web::get().to(graph::index)),
             )
+            .service(
+                actix_web::web::resource(&format!("{}/graph", app_prefix))
+                    .route(actix_web::web::get().to(graph::index)),
+            )
             .service(
                 actix_web::web::resource(&format!("{}/v1/openapi", app_prefix))
```
Comment (wking) [inline policy-engine/src/main.rs]: Hmm, seems like we'd want to adjust this one too?

File: policy-engine/src/graph.rs [hunk 2]
```diff
@@ -22,9 +22,14 @@ lazy_static! {
...
+    static ref GRAPH_INCOMING_REQS: Counter = Counter::new(
+        "graph_incoming_requests_total",
+        "Total number of incoming HTTP client request to /graph"
+    )
+    .unwrap();
     // Histogram with custom bucket values for serving latency metric (in seconds), values are picked based on monthly data
-    static ref V1_GRAPH_SERVE_HIST: Histogram = Histogram::with_opts(histogram_opts!(
-        "v1_graph_serve_duration_seconds",
+    static ref GRAPH_SERVE_HIST: Histogram = Histogram::with_opts(histogram_opts!(
+        "graph_serve_duration_seconds",
```
Comment (wking) [inline policy-engine/src/graph.rs:28]: Are we punting on [adding the `uri` and `version` labels][1] for now?

[1]: https://github.com/openshift/enhancements/pull/870/files#diff-694542247797fdac10c5f5ed4815621ac86992cd4082a7574aba201fdf5c9032R92

Comment (PratikMahajan) [reply policy-engine/src/graph.rs:28]: fixed

File: graph-builder/src/graph.rs [hunk 3]
```diff
@@ -62,9 +64,10 @@ lazy_static! {
         vec![5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 15.0, 20.0, 30.0 ]
     ))
     .unwrap();
-    static ref V1_GRAPH_INCOMING_REQS: Counter = Counter::new(
-        "v1_graph_incoming_requests_total",
-        "Total number of incoming HTTP client request to /graph"
+    static ref GRAPH_INCOMING_REQS: IntCounterVec = IntCounterVec::new(
+        Opts::new("graph_incoming_requests_total",
+        "Total number of incoming HTTP client request to /graph"),
```
Comment (wking) [inline graph-builder/src/graph.rs]: Drop `to /graph`?  And expect folks to notice the `endpoint` label to understand the actual target.

File: graph-builder/src/graph.rs [hunk 4]
```diff
@@ -98,12 +101,27 @@ pub fn register_metrics(registry: &prometheus::Registry) -> Fallible<()> {
...
+}
+
+async fn _index(
+    req: HttpRequest,
+    app_data: actix_web::web::Data<State>,
 ) -> Result<HttpResponse, GraphError> {
     let span = get_tracer().start("index");
     let _active_span = mark_span_as_active(span);
 
-    V1_GRAPH_INCOMING_REQS.inc();
```
Comment (wking) [inline graph-builder/src/graph.rs:105]: nit: I'd expect you could pull the endpoint off `req` without sharding out into `index` and `v1_index`, but I don't have a problem with sharding if you prefer that approach.

File: policy-engine/src/graph.rs [hunk 5]
```diff
@@ -12,19 +12,20 @@ use opentelemetry::{
...
 use serde_json;
 use std::collections::HashMap;
 
 lazy_static! {
-    static ref V1_GRAPH_INCOMING_REQS: Counter = Counter::new(
-        "v1_graph_incoming_requests_total",
-        "Total number of incoming HTTP client request to /graph"
+    static ref GRAPH_INCOMING_REQS: IntCounterVec = IntCounterVec::new(
+        Opts::new("graph_incoming_requests_total",
+        "Total number of incoming HTTP client request to /graph"),
```
Comment (wking) [inline policy-engine/src/graph.rs]: `to /graph`, like [this thread][1].

[1]: https://github.com/openshift/cincinnati/pull/537#discussion_r718784762

File: graph-builder/src/graph.rs (hunk 4 above)
Comment (PratikMahajan) [reply graph-builder/src/graph.rs:105]: fixed. pulling the endpoint information from request

File: commons/src/errors.rs [hunk 6]
```diff
@@ -18,11 +18,8 @@ pub mod prelude {
 pub use prelude::*;
 
 lazy_static! {
-    static ref V1_GRAPH_ERRORS: IntCounterVec = IntCounterVec::new(
-        Opts::new(
-            "v1_graph_response_errors_total",
-            "Error responses on /v1/graph"
-        ),
+    static ref GRAPH_ERRORS: IntCounterVec = IntCounterVec::new(
+        Opts::new("graph_response_errors_total", "Error responses on /graph"),
```
Comment (wking) [inline commons/src/errors.rs]: This will be for both graph endpoints now, right?  Can we drop `on /graph`?  Or replace it with "for graph endpoints"?

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**